src/meta/src/dashboard/prometheus.rs

```diff
@@ -92,7 +93,7 @@ pub async fn list_prometheus_cluster(
             .map(PrometheusVector::from)
             .collect();
         let memory_query =
-            "avg(process_resident_memory_bytes{job=~\"compute|meta|frontend\"}) by (job,instance)";
+            format!("avg(process_resident_memory_bytes{{job=~\"compute|meta|frontend\",{}}}) by (job,instance)", srv.prometheus_selector);
```

Comment:
Is trailing comma okay?

Reply:
Yes. 

- ✅ `metric_name{}`
- ✅ `metric_name{foo="bar"}`
- ✅ `metric_name{foo="bar",}`
- ❌ `metric_name{,}`
- ❌ `metric_name{foo="bar",,}`